scripts/test-e2e-standalone.sh

```diff
@@ -50,9 +50,13 @@ function finish() {
     echo Exit trap: killing OLS server
     kill %1
     rm -rf "$TMPDIR"
+    rm -rf "RAG_TMP_DIR"
```

Comment:
$RAG_TMP_DIR ?

Reply:
if you look at the top of the script it creates two tmpdirs for holding content, this ensures we clean both of them up.

Reply:
I mean isn't `$` missing there?

Reply:
ah, yes, thanks!